runtime/js/90_deno_ns.js

```diff
@@ -131,7 +131,7 @@ const denoNs = {
     internals.warnOnDeprecatedApi(
       "Deno.read()",
       new Error().stack,
-      "Use `reader.read()` instead.",
+      "See the Deno 1.x to 2.x Migration Guide for migration instructions at https://docs.deno.com/runtime/manual/advanced/migrate_deprecations",
```

Comment:
```suggestion
      "See the Deno 1 to 2 Migration Guide for more information at https://docs.deno.com/runtime/manual/advanced/migrate_deprecations",
```

Reply:
That said - this should be added directly to `warnOnDeprecatedApi` and the current suggestions should be left as they are